pkgs/by-name/le/letsdns/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+let
+  version = "1.2.1";
+in
+python3Packages.buildPythonApplication {
+  pname = "letsdns";
+  inherit version;
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "LetsDNS";
+    repo = "letsdns";
+    tag = version;
+    hash = "sha256-TwGVm7sEOPvUqtvaAuIU/X5W3H4VAC8dskNunt8UO0I=";
+  };
+
+  build-system = [
+    python3Packages.setuptools
+  ];
+
+  nativeCheckInputs = [
+    python3Packages.pytestCheckHook
+    versionCheckHook
+  ];
+
+  dependencies = with python3Packages; [
+    cryptography
+    dnspython
+    requests
+  ];
+
+  disabledTestPaths = [
+    # These tests require private upstream certificates
+    "tests/test_action.py"
+  ];
+
+  env = {
+    UNITTEST_CONF = "tests/citest.conf";
+  };
+
```

optional, but makes update process more automated
```suggestion

  passthru.updateScript = nix-update-script { };

```

---

@awwpotato Interesting, I'll need to read up on this feature. The first few search results seem to refer to [this project](https://github.com/Mic92/nix-update/). Is that what you suggest me looking into?